httpclient5/src/test/java/org/apache/hc/client5/http/impl/io/TestPoolingHttpClientConnectionManager.java

```diff
@@ -307,10 +308,10 @@ void testProxyConnectAndUpgrade() throws Exception {
         Mockito.when(conn.isOpen()).thenReturn(false);
         Mockito.when(future.get(1, TimeUnit.SECONDS)).thenReturn(entry);
         Mockito.when(pool.lease(
-                Mockito.eq(route),
-                Mockito.eq(null),
-                Mockito.any(),
-                Mockito.eq(null)))
+                        Mockito.eq(route),
```

Comment:
@arturobernalg Can we avoid formatting changes in the same commit with functional ones? If you want to change formatting please do it through the style check or at least in a separate commit.